activation/activation.go

```diff
@@ -770,7 +770,6 @@ func SignAndFinalizeAtx(signer *signing.EdSigner, atx *types.ActivationTx) error
 		Version: types.AtxV1,
 		Blob:    codec.MustEncode(wireAtx),
 	}
-	atx.Signature = wireAtx.Signature
```

Comment:
Now without this function actually signing the ATX I think every call to `SignAndFinalizeAtx` could just be replaced by:

```go
atx.SmesherID = signer.NodeID()
atx.SetID(types.ATXID(types.RandomHash()))
```

wdyt?